Resources/Prototypes/_Mono/Entities/ShipShield/shield.yml

```
@@ -0,0 +1,109 @@
+- type: entity
+  parent: BaseMachinePowered
+  id: GridShieldGenerator
+  name: ship shield generator
+  description: A machine that generates a shield around a ship.
+  components:
+  - type: Fixtures
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)